### DISCUSSION TITLE
Blockman (1.7.7) not working in VSCode (1.94.0-insider)

### DISCUSSION THREAD

**Discussion by n1colasf** [leodevbro/vscode-blockman#145] [Q&A] [answered]
Hi everyone! I’m encountering an issue with the extension as I’m unable to initialize it, and I’m getting the error shown in the screenshot. The version I’m using is 1.7.7, and my VSCode version is 1.94.0-insider. I followed the instructions provided in the tool’s details, but when I run Blockman Toggle Enable/Disable or any other command from the tool, it throws an error saying it cannot find the command.

Any idea what might be happening?
Thank you so much in advance for your help! I love this tool, and I miss it because it’s super useful! I hope you can assist me with this.

<img width="1624" alt="Captura de pantalla 2024-09-24 a la(s) 09 01 10" src="https://github.com/user-attachments/assets/3725bff2-dbd9-49a5-8b7b-366d384c3dbc">

**Comment by leodevbro** [comment]
Seems like a serious break after latest VS Code update. Both `VS Code` and `VS Code Insiders` broke things on the latest update.

![image](https://github.com/user-attachments/assets/63eae66b-fe0b-4973-be9f-5adda84b4e07)

```
mainThreadExtensionService.ts:107

Activating extension 'leodevbro.blockman' failed: Cannot find module 'c:\Users\leodevbroUser\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\vscode-textmate'
Require stack:
- c:\Users\leodevbroUser\.vscode\extensions\leodevbro.blockman-1.7.7\out\bracketAlgos\textMateLoader.js
- c:\Users\leodevbroUser\.vscode\extensions\leodevbro.blockman-1.7.7\out\bracketAlgos\settings.js
- c:\Users\leodevbroUser\.vscode\extensions\leodevbro.blockman-1.7.7\out\bracketAlgos\documentDecorationManager.js
- c:\Users\leodevbroUser\.vscode\extensions\leodevbro.blockman-1.7.7\out\extension.js
- c:\Users\leodevbroUser\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js.
```


I'll try to investigate it and/or inform Microsoft VS Code team about this problem.

**Reply by n1colasf** [reply]
Thank you for your quick response! 
I look forward to a fix so I can start using this great tool again. 
Big thanks for all the hard work!! 💪

**Reply by leodevbro** [reply]
https://github.com/microsoft/vscode/issues/230432

**Comment by leodevbro** [comment, marked as answer]
@n1colasf , I think I fixed it now with this commit on Blockman:

https://github.com/leodevbro/vscode-blockman/commit/f6ee335be12d485a2745985695c5eabc140d3d3d

New version of Blockman is v1.7.8 (2024-10-04 Tbilisi time).
Please update Blockman.


Thanks for reporting this problem quickly to me.

**Reply by n1colasf** [reply]
Yayyy! It's working correctly again! Thank you so much!! And congrats on creating such a great tool! 🚀🙌
